examples/proptypes-swagger-yml.js

```diff
@@ -0,0 +1,33 @@
+/* eslint no-use-before-define: 0 */
+import PropTypes from 'prop-types';
+
+export const EnumDefinitionPropTypes = PropTypes.oneOf(["default","special"]);
+
+export const ObjectDefinitionPropTypes = {
+	href: PropTypes.string,
+	text: PropTypes.string,
+};
+
+export const ArrayDefinitionPropTypes = PropTypes.arrayOf(PropTypes.string);
+
+export const DemoObjectPropTypes = {
+	inlineEnum: PropTypes.oneOf(["default","special"]),
+	inlineEnumRequired: PropTypes.oneOf(["default","special"]).isRequired,
+	inlineObject: PropTypes.shape({
+		href: PropTypes.string,
+		text: PropTypes.string,
+	}),
+	inlineObjectRequired: PropTypes.shape({
+		href: PropTypes.string,
+		text: PropTypes.string,
+	}).isRequired,
+	inlineArray: PropTypes.arrayOf(PropTypes.string),
+	inlineArrayRequired: PropTypes.arrayOf(PropTypes.string).isRequired,
+	arrRequired: PropTypes.arrayOf(PropTypes.string),
+	refEnum: EnumDefinitionPropTypes,
+	refEnumRequired: EnumDefinitionPropTypes.isRequired,
+	refObject: PropTypes.shape(ObjectDefinitionPropTypes),
+	refObjectRequired: PropTypes.shape(ObjectDefinitionPropTypes).isRequired,
+	refArray: ArrayDefinitionPropTypes,
+	refArrayRequired: ArrayDefinitionPropTypes.isRequired,
+};
```

Comment:
This is out of scope for this PR, but a general comment: Since we have those files as target output, why don't we whip up a quick test with jest and snapshots? It's the ideal use case for snapshots :)

Reply:
Nice idea: https://github.com/unic/openapi-proptypes-generator/issues/14